cmd/govim/gopls_client.go

```diff
@@ -19,31 +19,56 @@ const (
 var _ protocol.Client = (*govimplugin)(nil)
 
 func (g *govimplugin) ShowMessage(ctxt context.Context, params *protocol.ShowMessageParams) error {
-	g.logGoplsClientf("Asked to show a message: %v", params.Message)
+	g.logGoplsClientf("ShowMessage callback: %v", params.Message)
+	if params.Type != protocol.Error {
+		return nil
+	}
+
+	g.Schedule(func(govim.Govim) error {
+		opts := make(map[string]interface{})
+		opts["mousemoved"] = "any"
+		opts["moved"] = "any"
+		opts["padding"] = []int{0, 1, 0, 1}
+		opts["wrap"] = "false"
```

Comment:
This should be:

```
opts["wrap"] = false
```

What we send from Go will be JSON-marshaled. Then on the other end, Vim will translate from the JSON value to the appropriate Vim value (and fail it can't).

VimScript does not have true boolean values: see `:help TRUE` for more details. `v:true` and `v:false` do exist as convenience variables.

So my example in VimScript from earlier was actually incorrect, it should have read:

```vim
call popup_create("we have a problem", {"mousemoved":"any", "moved":"any", "padding":[0,1,0,1], "wrap": v:false, "close":"click", "border":[], "highlight": "ErrorMsg", "line":1})
```

Reply:
@myitcv I have fixed this in #519 https://github.com/dragfire/govim/blob/show_message_popup/cmd/govim/gopls_client.go#L32